shell/platform/windows/text_input_plugin.h

```diff
@@ -19,12 +20,28 @@ namespace flutter {
 
 class FlutterWindowsView;
 
+// A rectangle with a position and extent.
+//
+// Used to store the current composing rectangle when using multi-step (IME)
+// text input.
+struct Rect {
+  Rect() = default;
+  Rect(const Rect& rect) = default;
+  Rect& operator=(const Rect& other) = default;
+
+  double x = 0.0;
+  double y = 0.0;
+  double width = 0.0;
+  double height = 0.0;
+};
```

Comment:
We should either put this somewhere general (so that it's easy to use generally... maybe with some Point and Size friends? 😁 ), or make it private within the class; having the very generic `flutter::Rect` be tied specifically to the text input plugin class is weird.

Reply:
Done.